app/views/videos/index.html.haml

```diff
@@ -2,7 +2,7 @@
   %header
     %h3= link_to @categories[0].name, category_path(@categories[0])
   .videos.row
-    - @categories[0].videos.each do |video|
+    - @categories[0].recent_videos.each do |video|
```

Comment:
You have hard-coded all 3 videos :dancer: 

You should use `#each` loop - I strongly recommend to watch solution and rework whole template as you hardcode so much stuff.